xlml/utils/gpu.py

```diff
@@ -168,7 +188,10 @@ def create_resource_request(
     machine_type = accelerator.machine_type
     image = get_image_from_family(project=image_project, family=image_family)
     disk_type = f"zones/{gcp.zone}/diskTypes/pd-ssd"
-    disks = [disk_from_image(disk_type, 100, True, image.self_link)]
+    disks = [disk_from_image(disk_type, 1000, True, image.self_link)]
```

Comment:
Thanks for the change! Two questions:

1) is 1000GB needed for the case? it seems quite large, as needs by each accelerator.
2) Previously missed this hardcode `100` here. Could you help update method to below, so we could know it's a default value in helper function.

```
def disk_from_image(
    disk_type: str,
    boot: bool,
    source_image: str,
    disk_size_gb: int = 100,
    auto_delete: bool = True,
) -> compute_v1.AttachedDisk:
```

Reply:
We will get "failed to register layer: write /usr/lib/x86_64-linux-gnu/libnvinfer_static.a: no space left on device" with only 100 gb so I make it as a optional input to GPU. Thanks!